### PR TITLE
Pi

### PR DESCRIPTION
**Pull request by sublimino**

### CODE REVIEW

File: 01-Harbor.md [hunk 1]
```diff
@@ -41,18 +56,72 @@ certutil -addstore -f "ROOT" harbor-ca.crt
...
+
 ### Configure Harbor project to use Clair
+Turn on image scanning for the default library project by selecting the `library` project, clicking the `Configuration` tab. Select `Prevent vulnerable images from running` and set the threshold to high. This will prevent vulnerable images with one or more `high` vulnerability CVS's from getting pulled from this registry. Set the option to scan on push.
 
-Head to `https://192.168.99.100:30003/harbor/sign-in` to login to the Harbor UI. Username is `admin`, password is `kubecon1234`.
+Now build the image to be scanned. Clone the demo-API https://github.com/lukebond/demo-api onto your local machine and check out the `kubecon` branch.
 
-Turn on image scanning for the default library project by selecting the `library` project, clicking the `Configuration` tab, !!! TODO: @Andy/Pi decide how we want to use Clair, then fill this in !!!
+```bash
+git clone https://github.com/lukebond/demo-api.git
```
Comment (liamawhite) [inline 01-Harbor.md]: Can we just put this stuff into this repo?

File: 01-Harbor.md [hunk 2]
```diff
@@ -41,18 +56,72 @@ certutil -addstore -f "ROOT" harbor-ca.crt
...
+```bash
+docker build . -t <Minikube_IP>:30003/library/demo-api:vulnerable
+docker push <Minikube_IP>:30003/library/demo-api:vulnerable
+```
+
+In the Harbor UI, enter the library project and view the demo-api image. The vulnerability rate is set to HIGH.
+
+### Deploy vulnerable image
+
+Edit the setup/demo-api.yaml file by replacing your Enter the workshop repo and attempt to deploy the image from the registry into your minikube cluster.
```
Comment (liamawhite) [inline 01-Harbor.md]: I think you are missing some words here?

File: 01-Harbor.md (hunk 1 above)
Comment (sublimino) [reply 01-Harbor.md]: Will do after I've QA'd these steps

File: 01-Harbor.md (hunk 2 above)
Comment (sublimino) [reply 01-Harbor.md]: Removed erroneous phrase in master post-merge

**Pull request merged**